tool/gravity/cli/system.go

```diff
@@ -1074,6 +1074,20 @@ func systemUninstall(env *localenv.LocalEnvironment, confirmed bool) error {
 		return trace.Wrap(err)
 	}
 
+	// see if there are any mounts under the state directory
+	mounts, err := svm.ListMounts()
```

Comment:
Can this be encapsulated in a function called `UninstallMounts`?

Reply:
You mean as a part of `SystemManager` interface? I don't think so - as you can see below the logic is specific to the particular Gravity mount and the system manager is pretty generic.

Reply:
No, I meant moving the bits you added into a separate function and calling it here. Maybe my suggestion looking like exported function tripped you off.